akka-docs/src/main/paradox/java/stream/stream-dynamic.md

```diff
@@ -1,128 +1 @@
-a new subscriber. If there are no subscribers attached to this hub then it will not drop any elements but instead
-backpressure the upstream producer until subscribers arrive. This behavior can be tweaked by using the combinators
-`.buffer` for example with a drop strategy, or just attaching a subscriber that drops all messages. If there
-are no other subscribers, this will ensure that the producer is kept drained (dropping all elements) and once a new
-subscriber arrives it will adaptively slow down, ensuring no more messages are dropped.
-
-### Combining dynamic stages to build a simple Publish-Subscribe service
-
-The features provided by the Hub implementations are limited by default. This is by design, as various combinations
-can be used to express additional features like unsubscribing producers or consumers externally. We show here
-an example that builds a `Flow` representing a publish-subscribe channel. The input of the `Flow` is
-published to all subscribers while the output streams all the elements published.
-
-First, we connect a `MergeHub` and a `BroadcastHub` together to form a publish-subscribe channel. Once
-we materialize this small stream, we get back a pair of `Source` and `Sink` that together define
-the publish and subscribe sides of our channel.
-
-@@snip [HubDocTest.java]($code$/java/jdocs/stream/HubDocTest.java) { #pub-sub-1 }
-
-We now use a few tricks to add more features. First of all, we attach a `Sink.ignore`
-at the broadcast side of the channel to keep it drained when there are no subscribers. If this behavior is not the
-desired one this line can be simply dropped.
-
-@@snip [HubDocTest.java]($code$/java/jdocs/stream/HubDocTest.java) { #pub-sub-2 }
-
-We now wrap the `Sink` and `Source` in a `Flow` using `Flow.fromSinkAndSource`. This bundles
-up the two sides of the channel into one and forces users of it to always define a publisher and subscriber side
-(even if the subscriber side is just dropping). It also allows us to very simply attach a `KillSwitch` as
-a `BidiStage` which in turn makes it possible to close both the original `Sink` and `Source` at the
-same time.
-Finally, we add `backpressureTimeout` on the consumer side to ensure that subscribers that block the channel for more
-than 3 seconds are forcefully removed (and their stream failed).
-
-@@snip [HubDocTest.java]($code$/java/jdocs/stream/HubDocTest.java) { #pub-sub-3 }
-
-The resulting Flow now has a type of `Flow[String, String, UniqueKillSwitch]` representing a publish-subscribe
-channel which can be used any number of times to attach new producers or consumers. In addition, it materializes
-to a `UniqueKillSwitch` (see [UniqueKillSwitch](#unique-kill-switch)) that can be used to deregister a single user externally:
-
-@@snip [HubDocTest.java]($code$/java/jdocs/stream/HubDocTest.java) { #pub-sub-4 }
\ No newline at end of file
+../../scala/stream/stream-dynamic.md
```

Comment:
(Like in the other PR's, this still needs some work to make it a 'real' symlink)